Source/AIExtensions/Private/StateTree/Tasks/AIExtStateTreeTaskEnableActorCollision.cpp

```diff
@@ -0,0 +1,17 @@
+#include "StateTree/Tasks/AIExtStateTreeTaskEnableActorCollision.h"
+
+#include <StateTreeExecutionContext.h>
+
+EStateTreeRunStatus FAIExtStateTreeTaskEnableActorCollision::EnterState( FStateTreeExecutionContext & context, const FStateTreeTransitionResult & transition ) const
+{
+    const auto & instance_data = context.GetInstanceData< FInstanceDataType >( *this );
```

Comment:
use `TRACE_CPUPROFILER_EVENT_SCOPE_STR( __FUNCTION__ );` in each function of each task

Reply:
Done